frontend/src/components/AddDiscoveryTags.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import SelectableChipList from "./SelectableChipList";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: 'flex',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+
+        '& > *': {
```

Comment:
what is this for?

Reply:
Material UI ... it said to use this in order to use the component 😅